app/src/main/java/org/mozilla/focus/browser/LocalizedContent.java

```diff
@@ -110,6 +114,43 @@ private static void loadRights(@NonNull final WebView webView) {
         webView.loadDataWithBaseURL("file:///android_asset/rights.html", data, "text/html", "UTF-8", null);
     }
 
+    /**
+     * Load the content for focus:debug
+     */
+    private static void loadDebug(@NonNull final WebView webView) {
+        final Context context = webView.getContext();
+        final Resources resources = Locales.getLocalizedResources(context);
+
+        final Map<String, String> substitutionMap = new ArrayMap<>();
+        final String userAgentString = webView.getSettings().getUserAgentString();
+
+        final String release = Build.VERSION.RELEASE;
+        final int sdkVersion = Build.VERSION.SDK_INT;
+        final String baseOS = Build.VERSION.BASE_OS;
+        final String codename = Build.VERSION.CODENAME;
+        final String androidVersion = "Android SDK: " + sdkVersion + " (" + release +") | Base OS: " + baseOS + " | Codename: " + codename;
```

Comment:
I don't think we need anything in this androidVersion string because the user agent includes the SDK Version and it might be confusing for people filing bugs to copy so much information, so let's keep it as small as possible and remove these.